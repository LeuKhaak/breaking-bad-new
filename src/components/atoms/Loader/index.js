import React from "react";
import * as styles from "./styles.module.scss";
import loader from "../../../assets/images/loader.gif";

function Loader() {
  return (
    <div className={styles.loader}>
      <img className={styles.gif} src={loader} alt="#" />
    </div>
  );
}

export default Loader;
