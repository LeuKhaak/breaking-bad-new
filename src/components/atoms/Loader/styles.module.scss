@import "../../../assets/styles/styles.scss";
.loader {
  min-height: calc(100vh - 10rem);
  padding-top: 10rem;
  display: flex;
  justify-content: center;
  .gif {
    width: 10rem;
    height: 10rem;
  }
}
