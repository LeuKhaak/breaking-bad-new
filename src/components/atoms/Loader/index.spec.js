import React from "react";
import { shallow } from "enzyme";
import Loader from ".";

let component = null;
const setComponent = () => shallow(<Loader />);
beforeEach(() => {
  component = setComponent();
});

describe("Loader testing:", () => {
  test("is Loader rendered", () => {
    component = setComponent();
    expect(component).toMatchSnapshot();
  });
});
