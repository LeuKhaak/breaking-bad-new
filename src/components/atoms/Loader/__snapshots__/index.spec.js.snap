// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader testing: is Loader rendered 1`] = `
<div>
  <img
    alt="#"
    src="test-file-stub"
  />
</div>
`;
